docs/content/rc.md

```diff
@@ -426,7 +426,7 @@ Eg
     }
 
 
-If the rate parameter is not suppied then the bandwidth is queried
+If the rate parameter is not supplied then the bandwidth is queried
```

Comment:
This is an autogenerated part of file - the original is in `fs/accounting/token_bucket.go`

Reply:
Sorry, I didn't notice that since the `.md`. file is not under `content/commands`.

I regenerated the patch. I didn't revert the change in the `.md` file. Presumably it doesn't matter since it's machine generated anyway.

Reply:
> Sorry, I didn't notice that since the `.md`. file is not under `content/commands`.

Maybe I should put that in the docs too!

> I regenerated the patch. I didn't revert the change in the .md file. Presumably it doesn't matter since it's machine generated anyway.

Perfect